src/core/client/utility/missionText.ts

```diff
@@ -0,0 +1,16 @@
+import * as alt from 'alt-client';
+import * as native from 'natives';
+import { SYSTEM_EVENTS } from '../../shared/enums/system';
+/**
+ * Draw mission text on the bottom of screen
+ * @param  {string} text
+
+ */
+export function drawMissionText(text: string) {
+    native.clearPrints();
+    native.beginTextCommandPrint('STRING');
+    native.addTextComponentSubstringPlayerName(text);
+    native.endTextCommandPrint(100 * text.length, true);
```

Comment:
This native supports duration. Tested with 60s (60000ms) and passing duration should be something added.